docs/en/operation/Scalability-Tuning.md

```diff
@@ -73,10 +75,10 @@ ALLUXIO_WORKER_JAVA_OPTS+=" -Xms4g -Xmx4g"
 The frequency in which a worker checks in with the master is set by the following properties:
```

Comment:
nit: not related to this PR, but it should be "The frequency _with_ which" and "properties" to "property"

Reply:
done